### ISSUE TITLE
Not accept url action under tap_action of entities

### ISSUE THREAD

**Issue by valepe** [keesschollaart81/vscode-home-assistant#296]
After reading 1 & 2 👆, type your issue here and please include the related YAML code like this:

from version 0.100 a new url action is allowed for tap_action, hold_action and double_tap_action options of entities.

```YAML
      - entity: binary_sensor.myentity
        name: Name
        double_tap_action:
          action: url
          url_path: https://www.apple.com
```

and/or point to a repository containing your Home Assistant Configuration

**Comment by keesschollaart81** [comment]
Do you get an error or would you consider this a feature request for schema's around the binary sensors (as there are currently nog scheme's around it)

**Comment by valepe** [comment]
Vscode highlights it as errore, I suggest a schema update

**Comment by frenck** [comment]
Fixed in 1.7.0

**Comment by valepe** [comment]
Still 'Value is not accepted. Valid values: "toggle".' 
The property is action for a double_tap_action entity.
The strange is that same text block is also in another file and vscode doesn't show up the error.
The offending file is named 'alarms.yaml'. Does it matters?
